CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.4.2 (January 17th, 2024)
+
+* Limit error resets for misbehaving connections.
+* Fix selecting MAX_CONCURRENT_STREAMS value if no value is advertised initially.
+* Document that the default value of initial_max_send_streams is 100.
```

Comment:
I know there's a commit that says it did this, but it didn't end up doing it XD

Reply:
Just removed. The commit ended up just fixing formatting of the docs.

Reply:
Done